compiler/src/iree/compiler/Codegen/WGSL/WGSLReplacePushConstants.cpp

```diff
@@ -124,13 +133,18 @@ class WGSLReplacePushConstantsPass
       alignmentAttr = constantLoadOps[0].getAlignmentAttr();
     }
 
+    // We could store into a tensor<Nxi32>, but vec4s are better supported, so
+    // we'll use tensor<Nxvector<4xi32>> instead.
+    uint64_t numberOfVec4s = maxConstantIndex / 4 + 1;
```

Comment:
llvm has `llvm::divideCeil` for it. So if it's a perfect multiple of 4, we can avoid the `+1` there.

Reply:
Oh, did I make an off by one error? Hmm.. I think here if the "max index" is 4, that means I'd need [0, 1, 2, 3, 4], which is two vec4s?

Reply:
Ah you are right. This is index. Sorry I read too fast. :)

Reply:
No problem. I can add a comment too.